src/index.ts

```diff
@@ -0,0 +1,129 @@
+"use strict";
+
+import { nextTick } from "async/nextTick";
+import { debug } from "debug";
+import { dgram } from "dgram";
+import { EventEmitter } from "events";
+import { KadDHT } from "libp2p-kad-dht";
+import { multiaddr } from "multiaddr";
+import { PeerId } from "peer-id";
+import { PeerInfo } from "peer-info";
+const log = debug("discv5");
```

Comment:
something to consider, all libp2p modules have libp2p: prefix so you can enable all logs with libp2p:*

Reply:
I think we should simply integrate the winston logger here and remove the dependency on the logger currently in use.

Reply:
Usually libraries tend to use debug. Would be nice for libp2p-discv5 to comply with rest of libp2p libraries, don't know. @wemeetagain ?